### ISSUE TITLE
Editorial: Missing an embedded definition of Synchronized media in Guideline 1.2 understanding

### ISSUE THREAD

**Issue by maryjom** [w3c/wcag#2790]
This issue exists in both WCAG 2.1 and 2.2 Understanding Guideline 1.2: Time-based Media in the [Intent section](https://www.w3.org/WAI/WCAG22/Understanding/time-based-media#intent).

The 4th paragraph after the unordered list says, "Synchronized media is defined in the glossary as:" but the definition does not get incorporated into the content. 

This worked in WCAG 2.0, so when the formatting changed, the code to embed definitions stopped working.

**Comment by giacomo-petri** [comment]
It’s now functioning as intended: https://www.w3.org/WAI/WCAG22/Understanding/time-based-media#intent

<img width="1000" height="314" alt="screenshot showing the correct definition in place" src="https://github.com/user-attachments/assets/ac655db6-7b96-438e-92d1-9e39096cb2d9" />

> Synchronized media is defined in the glossary as: audio or video synchronized with another format for presenting information and/or with time-based interactive components, unless the media is a media alternative for text that is clearly labeled as such

Closing as already fixed
